modules/monitoring/manifests/network_checks.pp

```diff
@@ -0,0 +1,22 @@
+# == Class: monitoring::network_checks
+#
+# Sets up vDC gateways as separate hosts so they can become parents of child
+# hosts dependent on the state of their gateway.
+#
+define monitoring::network_checks (
+  $address,
+) {
+    icinga::host { $title:
+      host_name => $title,
+      hostalias => $title,
+      address   => $address,
```

Comment:
We need to pass in the `display_name` parameter here otherwise it will default to the `fqdn_short` of the monitoring machine (I'm becoming less sure about the `display_name` change as time goes on, still tempted to revert it).